themes/quire-starter-theme/layouts/data/search-index.html

```diff
@@ -58,7 +58,7 @@
   {{ .Scratch.Set "pageFullTitle" (.Scratch.Get "pageTitle") }}
 {{- end -}}
 
-{{- .Scratch.Set "pageData" (dict "id" $index "title" (.Scratch.Get "pageFullTitle") "url" (.Scratch.Get "pageURL") "type" (.Scratch.Get "pageType") "length" (.Scratch.Get "pageLength") "content" (.Scratch.Get "searchContent")) -}}
+{{- .Scratch.Set "pageData" (dict "id" $index "title" (.Scratch.Get "pageFullTitle") "url" (.Scratch.Get "pageURL" | urlize) "url1" (.Scratch.Get "pageURL" | absURL) "url2" (.Scratch.Get "pageURL" | relURL) "url3" (.Scratch.Get "pageURL" | safeURL) "type" (.Scratch.Get "pageType") "length" (.Scratch.Get "pageLength") "content" (.Scratch.Get "searchContent")) -}}
```

Comment:
were`url1`, `url2` and `url3` in there just for testing or are you actually using them all?

Reply:
Oops sorry about that ! I will take them out